### PR TITLE
Feature node js

### PR DESCRIPTION
**Pull request by JorgeRCV**

### CODE REVIEW

File: src/main/java/org/fundacion/jala/converter/core/NodeJsCompiler.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+            String resultOfExecution = null;
+            String result = "";
+            while ((resultOfExecution = bufferedReader.readLine()) != null) {
+                result += resultOfExecution + "\n";
+            }
+            return result;
+        } catch (IOException exception) {
+            LOGGER.error("Execute Exception" + exception.getLocalizedMessage());
+            return String.valueOf(new CompilerException(exception));
+        }finally {
```
Comment (Raymundogs) [inline src/main/java/org/fundacion/jala/converter/core/NodeJsCompiler.java]: It should be a space between } and finally.

File: src/main/java/org/fundacion/jala/converter/core/parameter/NodeJsParameter.java [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package org.fundacion.jala.converter.core.parameter;
+
+public class NodeJsParameter extends Parameter{
```
Comment (Raymundogs) [inline src/main/java/org/fundacion/jala/converter/core/parameter/NodeJsParameter.java]: Class should have javadoc.

File: src/main/java/org/fundacion/jala/converter/core/parameter/NodeJsParameter.java [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+
+    public NodeJsParameter(String newFilePath) {
```
Comment (Raymundogs) [inline src/main/java/org/fundacion/jala/converter/core/parameter/NodeJsParameter.java]: The variable should be final.

File: src/main/java/org/fundacion/jala/converter/core/parameter/NodeJsParameter.java [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+        super(newFilePath);
+    }
+    public NodeJsParameter() {
```
Comment (Raymundogs) [inline src/main/java/org/fundacion/jala/converter/core/parameter/NodeJsParameter.java:22]: There should be a space between constructors.

File: src/main/java/org/fundacion/jala/converter/core/parameter/NodeJsParameter.java (hunk 2 above)
Comment (Raymundogs) [inline src/main/java/org/fundacion/jala/converter/core/parameter/NodeJsParameter.java]: There should be a space between Parameter and {.

File: src/main/java/org/fundacion/jala/converter/core/NodeJsCompiler.java [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+ * This class compiles a Node.Js file.
+ */
+public class NodeJsCompiler {
+    private NodeJsParameter nodeJsParameter;
+    private BufferedReader bufferedReader;
+    private String nodeExe = "\"" + "thirdparty\\windows\\nodebin\\node-v16.0.0-win-x64\\node.exe" + "\"";
+    private static final Logger LOGGER = LogManager.getLogger();
+
+    /**
+     * Compiles a java file.
```
Comment (JessickaMoya) [inline src/main/java/org/fundacion/jala/converter/core/NodeJsCompiler.java]: Should be "nodeJs file" intead of "java file" in javadoc

File: src/test/java/org/fundacion/jala/converter/core/NodeJsCompilerTest.java [hunk 6]
```diff
@@ -0,0 +1,16 @@
+package org.fundacion.jala.converter.core;
+
+import org.fundacion.jala.converter.core.exceptions.CompilerException;
+import org.junit.Test;
+
```
Comment (JessickaMoya) [inline src/test/java/org/fundacion/jala/converter/core/NodeJsCompilerTest.java:15]: Delete blank space between imports

Comment (Raymundogs) [inline src/test/java/org/fundacion/jala/converter/core/NodeJsCompilerTest.java:15]: You should remove this empty line.

File: src/test/java/org/fundacion/jala/converter/core/NodeJsCompilerTest.java [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+import static org.junit.Assert.*;
```
Comment (Raymundogs) [inline src/test/java/org/fundacion/jala/converter/core/NodeJsCompilerTest.java]: You should import the specific library.

**Review by JessickaMoya — APPROVED** [review]

**Review by JRojas999 — APPROVED** [review]

**Review by Raymundogs — APPROVED** [review]

**Pull request merged**